### PR TITLE
OpenVINO tokenizers

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mryzhov** [comment]
I'm OK to be a maintainer of openvino-tokenizers-feedstock repo

**Comment by apaniukov** [comment]
I'm OK to be a maintainer of openvino-tokenizers-feedstock repo

**Comment by ilya-lavrenov** [comment]
@conda-forge/help-c-cpp this is ready for review

File: recipes/openvino-tokenizers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+{% else %}
+    {% set cmake_args = "$CMAKE_ARGS" %}
+{% endif %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/openvinotoolkit/openvino_contrib/archive/4c60cb1b20f45baef27b4e79fba30f929359987b.tar.gz
```
Comment (ilya-lavrenov) [inline recipes/openvino-tokenizers/meta.yaml:20]: it's an initial release of this component against latest OpenVINO 2023.2
Later, OpenVINO tokenizers will be aligned with OpenVINO and have coupled releases.

**Comment by sandye51** [comment]
@conda-forge/help-c-cpp please, help to review.

**Review by xhochy — APPROVED** [review]

**Pull request merged**